### PR TITLE
TS: Add support for type="vanished" as type="obsolete" for Qt 5

### PR DESCRIPTION
**Pull request by Wohlstand**
Issue #3901

### CODE REVIEW

File: translate/storage/ts2.py [hunk 1]
```diff
@@ -349,7 +350,7 @@ def merge(self, otherunit, overwrite=False, comments=True, authoritative=False):
             self.markfuzzy(False)
 
     def isobsolete(self):
-        return self._gettype() == "obsolete"
+        return self._gettype() == "obsolete" or self._gettype() == "vanished"
```
Comment (nijel) [inline translate/storage/ts2.py]: Can you please add test covering this? Apparently this method is not tested at all.

Comment (Wohlstand) [reply translate/storage/ts2.py]: Do you mean adding the test case into existing unit test to verify this feature? Ok, I'll add that once I'll reach my PC.

**Comment by nijel** [comment]
Merged, thanks for your contribution!

**Comment by Wohlstand** [comment]
Thanks! :fox_face: 
The last question for curiosity: is a new release planned in some time moment, or will happen once some internal milestones will be completed?

**Comment by nijel** [comment]
This is more question to @unho  and I'd like to know answer as well ;-)

**Comment by unho** [comment]
@Wohlstand We can certainly look into a new release soon since last one was in october and @nijel has been actually quite a good progress.

**Comment by unho** [comment]
@Wohlstand @nijel Translate Toolkit 2.4.0 is out!

**Pull request merged**